### ISSUE TITLE
IndexError for Multi-Person-Pose Estimation

### ISSUE THREAD

**Issue by Digbijay2** [spmallick/learnopencv#154]
I am running the code for multi-person pose estimation:
https://github.com/spmallick/learnopencv/blob/master/OpenPose-Multi-Person/multi-person-openpose.ipynb

but it shows an index error. Please help me to recover it.

IndexError                                Traceback (most recent call last)
<ipython-input-20-397a8a8683b9> in <module>()
----> 1 valid_pairs, invalid_pairs = getValidPairs(output)

<ipython-input-13-7ca5f7059574> in getValidPairs(output)
      9     for k in range(len(mapIdx)):
     10         # A->B constitute a limb
---> 11         pafA = output[0, mapIdx[k][0], :, :]
     12         pafB = output[0, mapIdx[k][1], :, :]
     13         pafA = cv2.resize(pafA, (frameWidth, frameHeight))

IndexError: index 31 is out of bounds for axis 1 with size 19

![aa](https://user-images.githubusercontent.com/37648166/47517936-45622c80-d8a7-11e8-9925-ad29cced86bb.png)

**Comment by vikasguptaiisc** [comment]
Which prototxt file are you using? Did you make any changes to the code?

I just ran the code and it runs fine.

**Comment by Digbijay2** [comment]
Vikas, actually i took the prototxt file from another repository but it looks exactly same. Do you think that the error occurred due to this??

**Comment by vikasguptaiisc** [comment]
Yes, In that file, the PAFs are commented out. Thus you are getting index error.

**Comment by Digbijay2** [comment]
So will i have to change the whole file or i can make changes in my existing prototxt??
Can you tell me the exact parameters and use of PAFs because i am unable to understand its use.

**Comment by vikasguptaiisc** [comment]
You will have to use the same file provided with the code. The PAFs are used to find the connections between limbs when multiple person are present. Everything is given in the blog. Please refer to that.

**Comment by Digbijay2** [comment]
Okay thanks for the help. Let me check it again

**Comment by Digbijay2** [comment]
yess..it works vikas. i have used another .prototxt file thats why i face index error. thanks for the help.
